data-structures/arrays/python/array_2D_update_constants.py

```diff
@@ -0,0 +1,36 @@
+# Raspberry Pi Foundation
+# Developed as part of Ada Computer Science
+# 
+# Usage licensed under CC BY-NC-SA 4.0
+
+
+# Index values of each level
+LEVEL1 = 0 
+LEVEL2 = 1
+LEVEL3 = 2
+
+
+def main():
+    # Declare and initialise the two-dimensional list of words
+    spelling_words = [
+        ["me", "do", "her", "it", "him"],
+        ["put", "ask", "says", "red", "any"],
+        ["they", "where", "friend", "fast", "class"]
+    ]
+
+    # Ask the user for the new word
+    new_word = input("Enter the new word: ")
+
+    # Update the last word for Level 3
+    word_index = 4
```

Comment:
so again here I would do something like: last = len(spelling_words[LEVEL3]) - 1 or maybe have more constants. I think we maybe need to thrash through the emphasis on the last word.